### DISCUSSION TITLE
FEBI boundary to user created object

### DISCUSSION THREAD

**Discussion by Abhi-SimuTech** [ansys/pyaedt#3914] [Q&A]
Hi,

How to assign FEBI region to a air box (user created) in a Hybrid modal n/w model. The "assign_hybrid_region" does not provide the option to assign a FEBI boundary! And, I do not wish to use "set_auto_open", it apparently changes solution type to HFSS type from Hybrid type. Could not find another command that will enable me to assign FEBI on the object!

Secondly, I was trying to follow the example in https://aedt.docs.pyansys.com/version/stable/examples/02-HFSS/Create_3d_Component_and_use_it.html#sphx-glr-examples-02-hfss-create-3d-component-and-use-it-py. but the 3D component does not have the boundary (FEBI) in it!

Any suggestions!

Regards

**Comment by Abhi-SimuTech** [comment]
hfss.create_open_region(Frequency='10.5GHz', Boundary='FEBI', ApplyInfiniteGP=False, GPAXis='-z')

component_path = os.path.join(os.getcwd(), generate_unique_name("component_test")+".a3dcomp")
my_modeler.create_3dcomponent(component_path, "my_antenna")

But, the hybrid region (as well as the vacuum RadiatingSurface) does not get included in the 3Dcomponent!

My goal is to have this antenna model (with FEBI box around) in another model that has another antenna with FEBI, and able to simulate this two antenna setup (with their individual FEBIs).

**Comment by Samuelopez-ansys** [comment]
@Abhi-SimuTech You are right the FEBI assignment is not implemented for this case. Please submit an issue, because it is not a Discussion.

Thanks,
Samuel
